docs: add note on default import path (#514)
antfu/unplugin-vue-components: README.md

}
</script>
```
> **Note**

> > that by default this plugin will import components from the `src/components` path. You can customize this using the `dirs` property in the plugin options.

Reviewer: ```suggestion
> By default this plugin will import components in the `src/components` path. You can customize it using the `dirs` option.
```
